readme.md

```diff
@@ -70,6 +70,25 @@ public class TestBaseSample  :
 <!-- endSnippet -->
 
 
+## xunit Fixture
+
+In addition to `XunitContextBase` class approach, one is also possible to use  `IContextFixture` to gain access to `XunitContext` :
+```cs
+public class UsingClassFixture : IContextFixture
```

Comment:
can u make this a snippet like the other code in the readme. 

see https://github.com/SimonCropp/MarkdownSnippets